AISKU/Tests/Unit/src/AISKUSize.Tests.ts

```diff
@@ -2,7 +2,7 @@ import { AITestClass, Assert } from "@microsoft/ai-test-framework";
 import * as pako from "pako";
 
 export class AISKUSizeCheck extends AITestClass {
-    private readonly MAX_DEFLATE_SIZE = 40;
+    private readonly MAX_DEFLATE_SIZE = 41;
```

Comment:
This is going to increase with each new beta feature -- once all of the features are added I'll then spend some time to try and reduce this again